cli/src/deno.rs

```diff
@@ -13,9 +13,11 @@ use deno_runtime::deno_core::{
 use deno_runtime::permissions::Permissions;
 use deno_runtime::worker::{MainWorker, WorkerOptions};
 use deno_runtime::{colors, BootstrapOptions};
+use tokio::fs;
+use url::{Host, Url};
 
 use crate::commands::extensions::api;
-use crate::commands::extensions::extension::ExtensionState;
+use crate::commands::extensions::extension::{extensions_path, ExtensionState};
```

Comment:
```suggestion
use crate::commands::extensions::extension::ExtensionState;
```

Should import on the module level here.